practice-app/api/tests.py

```diff
@@ -1,3 +1,13 @@
-from django.test import TestCase
+from django.http import response
+from django.test import TestCase, Client
 
-# Create your tests here.
+
+class GettingPlacesTestCase(TestCase):
+    def test_get_places(self):
+        c = Client()
+    
+        response = c.get('/api/joke/ani')
+        self.assertEqual(response.status_code, 200)
+
+        response2 = c.get('/api/joke/animal')
+        self.assertEqual(response2.status_code,200)
```

Comment:
Can you also test against a joke that is know to exist. Also, need to handle non-existent cases. What should happen for non 200 returning cases? Should be tested.
What does "Places" have to do with "jokes"?

Reply:
I'm updating the code right now. I have a question though. Should I create a new pull request or is there a way to edit this one to cover the new commit?